op-e2e/actions/proofs/isthmus_requests_test.go

```diff
@@ -0,0 +1,127 @@
+package proofs_test
+
+import (
+	"context"
+	"math/big"
+	"testing"
+
+	actionsHelpers "github.com/ethereum-optimism/optimism/op-e2e/actions/helpers"
+	"github.com/ethereum-optimism/optimism/op-e2e/actions/proofs/helpers"
+	"github.com/ethereum-optimism/optimism/op-e2e/e2eutils"
+	"github.com/ethereum/go-ethereum"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/core/types"
+	"github.com/ethereum/go-ethereum/params"
+	"github.com/holiman/uint256"
+	"github.com/stretchr/testify/require"
+)
+
+func TestIsthmusExcludedPredeploys(gt *testing.T) {
+	// Ensures that if EIP-6110, EIP-7251, or EIP-7002 predeploys are deployed manually after the fork,
```

Comment:
It looks like EIP-6110 is not covered in this test - should it be?

Reply:
no, 6110 isn't adding a new predeploy. I'll remove this comment